packages/gatsby-remark-responsive-iframe/src/index.js

```diff
@@ -32,9 +34,18 @@ module.exports = ({ markdownAST }, pluginOptions = {}) =>
         const height = iframe.attr(`height`)
 
         if (acceptedDimensions(width, height)) {
+          const existingStyle = $(`iframe`).attr(`style`) // Other plugins might set border: 0
+          // so we make sure that we maintain those existing styles. If other styles like height or
+          // width are already defined they will be overridden anyway.
+
+          let fullStyle = ``
+          if (existingStyle && needsSemicolon(existingStyle)) {
```

Comment:
Does this mean the `existingStyle` is ignored if it doesn't need a semicolon?

Reply:
Nice catch, I've amended my PR to address this and added a test case.